### ISSUE TITLE
Open Science Festival 2023

### ISSUE THREAD

**Issue by santoshilam** [delftopenhardware/openhardwarenl#10]
The third edition of the [open science festival](https://opensciencefestival.nl/) is scheduled to take place on the 31st of August at Erasmus university Rotterdam.

We have three avenues for submitting proposals (by 15th May), we can do a "Workshop" (what delft open hardware did in 2021), we can organize a "Session", get a booth at the "Marketplace". Of the three, marketplace seems to be the best fit in terms of having reach and getting a lot of eyeballs. It will allow us to set up a space explaining things about open hardware, display some projects and get people to be involved with what we are doing.

What do people think about with regard to the proposal for the festival, does the marketplace sound good?  
If yes, what elements do you think is needed there? 
I have created the page in my branch [here](https://github.com/delftopenhardware/openhardwarenl/blob/outline_strategy/opensciencefestival2023.md), the questions that need to be answered and other details are in details there.

**Comment by Vsaggiomo** [comment]
Agree, the marketplace seems to be the way to go. We have two easily portable and already known projects (syringe pumps and microscope).
